lib/matplotlib/pyplot.py

```diff
@@ -426,43 +426,43 @@ def figure(num=None,  # autoincrement if None, else integer from 1-N
         `num`.
 
     figsize : tuple of integers, optional, default: None
-        width, height in inches. If not provided, defaults to rc
-        figure.figsize.
+        width, height in inches. If not provided, defaults to 
+        :rc:`figure.figsize`.
 
     dpi : integer, optional, default: None
-        resolution of the figure. If not provided, defaults to rc figure.dpi.
+        resolution of the figure. If not provided, defaults to :rc:`figure.dpi`.
 
     facecolor :
-        the background color. If not provided, defaults to rc figure.facecolor.
+        the background color. If not provided, defaults to :rc:`figure.facecolor`.
 
     edgecolor :
-        the border color. If not provided, defaults to rc figure.edgecolor.
+        the border color. If not provided, defaults to :rc:`figure.edgecolor`.
 
     frameon : bool, optional, default: True
         If False, suppress drawing the figure frame.
 
-    FigureClass : class derived from matplotlib.figure.Figure
+    FigureClass : class derived from the :class:`~matplotlib.figure.Figure` class.
```

Comment:
FigureClass : subclass of \`~matplotlib.figure.Figure\`

should be sufficient.